swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        items = []
+        for item in self.inventory:
+            if item.category == category:
+                items.append(item)
+        return items
+
+
+    def swap_items(self, friend, my_item, their_item):
+        if my_item in self.inventory and their_item in friend.inventory:
+            friend.add(my_item)
+            self.add(their_item)
+            self.remove(my_item)
+            friend.remove(their_item)
+            return True
+
+    def swap_first_item(self, friend):
+        if self.inventory == [] or friend.inventory == []:
+            return False
+        else:
+            self.swap_items(friend, self.inventory[0], friend.inventory[0])
+            return True
```

Comment:
Since `swap_items()` up top also returns `True` after swapping in L28-31, we can shorten these two lines a little more:
```
return self.swap_items(friend, self.inventory[0], friend.inventory[0])
```